setup.cfg

```diff
@@ -30,7 +30,7 @@ install_requires =
     pyparsing>=2.2
     requests>=2.22
     roman_datamodels>=0.11.0
-    romanad==0.10.0
+    rad>=0.11.0
```

Comment:
Should we remove rad altogether since it comes with romand_datamodels. Is romancal using anything directly from rad or even importing it?

Reply:
Rad can be removed. It was there when we needed an earlier version of rad to pass the tests and is no longer needed.